owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/history/util/IDifferenceHandler.java

```diff
@@ -0,0 +1,15 @@
+package fr.openwide.core.jpa.more.business.history.util;
+
+import java.util.Date;
+
+import fr.openwide.core.jpa.more.business.difference.model.Difference;
+
+/**
+ * Met à jour un objet en fonction des différences constatées sur cet objet.
+ * <p>Se résume pour l'instant à simplement rafraîchir certains {@link HistoryEvent}.
+ */
+public interface IDifferenceHandler<T> {
```

Comment:
Well, the only implementation of this interface dealed with HistoryEvents, but this interface is not absolutely tied to it, it could be used for other data types.
Now, if I remember correctly, the problem with HistoryEvent was that you wanted to make "author" a HistoryValue, which could require some work on the project HistoryEvent comes from.
Maybe we should adress this in a different PR?